src/predefined-variables.ts

```diff
@@ -1,12 +1,17 @@
 import camelCase from "camelcase";
 import {GitData} from "./git-data";
+import {Utils} from "./utils";
 
 type PredefinedVariablesOpts = {
     gitData: GitData;
     argv: {unsetVariables: string[]};
 };
 
 export function init ({gitData, argv}: PredefinedVariablesOpts): {[name: string]: string} {
+    const CI_SERVER_URL = `https://${gitData.remote.host}:443`;
```

Comment:
This isn't working for me unfortunately. I think it's because this should be `${gitData.remote.port}` rather than the hard-coded `443`. Hopefully this will enable support for self-hosted instances on alternate ports.